signac/contrib/collection.py

```diff
@@ -417,7 +417,7 @@ class Collection:
 
     """
 
-    def __init__(self, docs=None, primary_key="_id", compresslevel=0, _trust=False):
+    def __init__(self, docs=None, primary_key="id_", compresslevel=0, _trust=False):
```

Comment:
I'm not sure if we want to rename the primary key here -- we may want to revert the changes to strings named `"_id"`. I'd like to consult @vyasr on this, since it was his suggestion to rename the variable.

Reply:
I agree, in strings like this I think we can leave this as is. I was primarily concerned with variables as @bdice indicated.